### DISCUSSION TITLE
findAllActiveListingsByShop by URL request - change from v2 to v3 not working

### DISCUSSION THREAD

**Discussion by iartp** [etsy/open-api#957] [Q&A] [answered]
Hello.

With api V2 I was using the following link to get active listings from my shop (fed into a FileMaker database).

**https://openapi.etsy.com/v2/shops/MY_SHOP_NAME/listings/active?api_key=MY_API_KEY&limit=1&offset=0&sort_order=down**

I've updated the link to v3 syntax (below) but it doesn't appear to work anymore

**https://openapi.etsy.com/v3/application/shops/MY_SHOP_ID_NUMBER/listings/active?api_key=MY_API_KEY&limit=1&offset=0&sort_order=down**

as I get the following error message:
**{"error":"Missing x-api-key header or client_id"}**

On the v2 the MY_SHOP_NAME was the name of the shop which I have changed to the SHOP_NUMBER of my shop for v3 as it states in the documentation. The MY_API_KEY is the same as the one I was previously using.

Any pointers greatly appreciated.

**Comment by Gareth064** [comment, marked as answer]
Unfortunately the v3 API of fundamentally different.

The API key needs to be added as a header now and does not form part of the URL.

This article describes how requests should be formed. https://developers.etsy.com/documentation/essentials/requests

**Reply by iartp** [reply]
Many thanks for the link - got it working now.
